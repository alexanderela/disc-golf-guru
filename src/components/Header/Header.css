.Header {
	align-items: center;
	/*background-color: #03ad03;*/
	background: url('../../assets/disc-golf-field-2.jpg');
	background-repeat: no-repeat;
  background-position: center;
  background-size: 110%;
  background-position-x: -5rem;
	color: white;
	display: flex;
	justify-content: center;
	height: 12rem;
	margin: 0;
	width: 100%;
	/*font-family: 'Pacifico', cursive;*/
	/*font-family: 'Open Sans', sans-serif;*/
	font-family: 'Holtwood One SC', serif;
	/*font-family: 'Sriracha', cursive;*/
}

.NavLink-site-title {
	text-decoration: none;
	color: #FFF;
}

.site-title {
	font-size: 3rem;
	background-color: rgba(255,255,255, 0);
	/*padding: 0 5rem 0 0;*/
	text-shadow: #000 5px 5px 5px;
}