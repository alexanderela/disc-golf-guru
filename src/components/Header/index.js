import React from 'react';
import './Header.css'

const Header = () => (
	<div className='Header'>
		<h1>Hella Golf Weather</h1>
	</div>
)

export default Header