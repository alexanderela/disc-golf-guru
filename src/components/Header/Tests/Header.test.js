import React from 'react';
import ReactDOM from 'react-dom';
import Header from '../';
import { shallow } from 'enzyme';

describe('Header', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<Header />)
	})
	it('should render like snapshot', () => {
		expect(wrapper).toMatchSnapshot()
	})
	
})