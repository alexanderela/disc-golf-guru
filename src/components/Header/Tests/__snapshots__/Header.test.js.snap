// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render like snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        aria-current="page"
        className="NavLink-site-title"
        to="/"
      >
        <h1
          className="site-title"
        >
          Disc Golf Guru
        </h1>
      </NavLink>,
      "className": "Header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "aria-current": "page",
        "children": <h1
          className="site-title"
        >
          Disc Golf Guru
        </h1>,
        "className": "NavLink-site-title",
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Disc Golf Guru",
          "className": "site-title",
        },
        "ref": null,
        "rendered": "Disc Golf Guru",
        "type": "h1",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          aria-current="page"
          className="NavLink-site-title"
          to="/"
        >
          <h1
            className="site-title"
          >
            Disc Golf Guru
          </h1>
        </NavLink>,
        "className": "Header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <h1
            className="site-title"
          >
            Disc Golf Guru
          </h1>,
          "className": "NavLink-site-title",
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Disc Golf Guru",
            "className": "site-title",
          },
          "ref": null,
          "rendered": "Disc Golf Guru",
          "type": "h1",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
